### DISCUSSION TITLE
Web Client won't pop up under OAuth 2.0 Client IDs

### DISCUSSION THREAD

**Discussion by ikerinagy** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#919] [Q&A]
I've followed through the setup steps, but It doesn't seem to pop up. Can I get some help here?

![image](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/assets/145727702/c436557a-cd9a-4906-ba2c-5490c78841cf)

**Comment by sumeshir26** [comment]
@ikerinagy You found the answer?

**Reply by ikerinagy** [reply]
yes, there is a "create credentials" button, and you need to add it from there manually

**Reply by fvazcoelho** [reply]
I followed the instructions in the video below, and with that, I was able to fix the problem.

https://www.youtube.com/watch?v=Jxog8xE2mDo&ab_channel=RandroTech
